autotest.pm

```diff
@@ -262,9 +254,7 @@ sub handle_sigterm {
     _exit(1);
 }
 
-sub start_process {
-    my $child;
-
+sub start_process ($child = undef) {
```

Comment:
Is it intended that `$child` can be passed as an argument now?

Reply:
No, that does not make sense. Thanks.